server/game/cards/04.1-BotK/Kudaka.js

```diff
@@ -1,10 +1,19 @@
 const DrawCard = require('../../drawcard.js');
 
 class Kudaka extends DrawCard {
-    setupCardAbilities(ability) { // eslint-disable-line no-unused-vars
+    setupCardAbilities(ability) {
+        this.reaction({
+            title: 'Gain 1 fate and draw 1 card',
+            limit: ability.limit.perRound(2),
+            effect: 'gain 1 fate and draw 1 card',
+            when: {
+                onClaimRing: (event, context) => event.conflict && event.conflict.ring.element === 'air' && event.player === context.player
```

Comment:
This will only explicitly check for the air ring - you need to check for any additional elements.
You should use event.conflict.hasElement('air'), I think

Reply:
Ah, i forgot that it counts any additional elements as being claimed. I'll add that and some tests tomorrow.